openedx/core/djangoapps/external_user_ids/docs/decisions/0001-externalid.rst

```diff
@@ -0,0 +1,40 @@
+Record External ID Table Decision
+=================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+Third Party's at times require a unique user ID as part of the data that they can
+access about a Learner. We would like to avoid sending them internal database IDs
+to protect our internal data models.
+
+Currently, we use the database row ID for each user internally. We also have Hash IDs
+that are used for some reporting and other external sources as part of the anonymous ID
+table.  We would like to avoid using the anonymous IDs because they have a specific use case.
+
+
+Decisions
+---------
+
+- External IDs will be UUIDs.
+- User's will only have exactly 1 external id per ExternalIDType.
+- All Types will have a clear description in the ExternalIDType table.
```

Comment:
Suggestion: add a why. Something like "so that we can later determine why this type was created. If the type was created to send to Company A, the description can help determine if ids of this type may also be sent to Company B".